app/defines/csr.h

```diff
@@ -37,8 +37,8 @@
 // sample relevant csr's for pmon
 void rvc_sample_csr(int *cycle_low, int *instret_low){
```

Comment:
@roman012285 - Still this function name makes no sense to me..
"rvc_sample_csr" -> what is this function actually do?
I feel like I'm missing something here

Reply:
we read the clycle and instret in alive_ipc test and calculates ipc